### PR TITLE
add syncer checklist

### PR DESCRIPTION
**Pull request by zier-one**
### What problem does this PR solve?
Add syncer checklist for mysql to be downstream

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/syncer-support.md [hunk 1]
```diff
@@ -48,3 +48,4 @@ Additionally, please provide the following info before submitting your issue. Th
 3. Common issues
     - [ ] Is the `worker-count` set to a reasonable value (suggest-32)?
     - [ ] Is the `batch` set to a reasonable value (suggest 100-1000)?
+    - [ ] Is the downstream server mysql and using multiStatements or not utf8mb4 charset (syncer can only provide limited support for MySQL as a downstream server)?
```
Comment (kennytm) [inline .github/ISSUE_TEMPLATE/syncer-support.md]: This is a bit hard to parse (`(a and b) or c`? `a and (b or c)`?). And I suppose `multiStatements` should be turned on, and the character set should be set to `utf8mb4`? Perhaps split into two items if there are two thing to check.

**Review by kennytm — APPROVED** [review]
LGTM (would be perfect if you fix the formatting (e.g. `mysql` → `MySQL`, `multiStatements` → `` `multiStatements` `` etc.))

**Comment by zier-one** [comment]
@GregoryIan PTAL

File: .github/ISSUE_TEMPLATE/syncer-support.md [hunk 2]
```diff
@@ -48,3 +48,5 @@ Additionally, please provide the following info before submitting your issue. Th
 3. Common issues
     - [ ] Is the `worker-count` set to a reasonable value (suggest-32)?
     - [ ] Is the `batch` set to a reasonable value (suggest 100-1000)?
+    - [ ] Is the downstream server MySQL and using `multiStatements` (syncer doesn't provide `multiStatements`)?
```
Comment (IANTHEREAL) [inline .github/ISSUE_TEMPLATE/syncer-support.md]: remove it, user don't know what's `multiStatements`, and we can support `multiStatements` in syncer

**Comment by IANTHEREAL** [comment]
LGTM
